### PR TITLE
Add build script for package php-encryption: v2.2.1, phpunit: 9.0.0, spdx-licenses: 1.5.5, php-webdriver: 1.4.7

### PR DESCRIPTION
**Pull request by vedang-wartikar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@Ryan-Dmello please review

File: p/php-encryption/php-encryption_ubi8.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: defuse/php-encryption
+# Version	: v2.2.1
+# Source repo	: https://github.com/defuse/php-encryption
+# Tested on	: RHEL 8.3
```
Comment (Ryan-Dmello) [inline p/php-encryption/php-encryption_ubi8.sh:6]: Distro information is not matching with file name

File: p/phpunit/phpunit_ubi8.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: phpunit/phpunit
+# Version	: 9.0.0
+# Source repo	: https://github.com/sebastianbergmann/phpunit
+# Tested on	: RHEL 8.3
```
Comment (Ryan-Dmello) [inline p/phpunit/phpunit_ubi8.sh:6]: Distro information is not matching with file name

File: s/spdx-licenses/spdx-licenses_ubi8.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# -----------------------------------------------------------------------------
+#
+# Package	: composer/spdx-licenses
+# Version	: 1.5.5
+# Source repo	: https://github.com/composer/spdx-licenses
+# Tested on	: RHEL 8.3
```
Comment (Ryan-Dmello) [inline s/spdx-licenses/spdx-licenses_ubi8.sh:6]: Distro information is not matching with file name

**Review by Ryan-Dmello — CHANGES_REQUESTED** [review]
Other than comments related to file name not matching distro information, changes looks ok

**Review by kandarpamalipeddi — APPROVED** [review]

**Pull request merged**